src/main/rx/jetiexbus.c

```diff
@@ -0,0 +1,595 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Authors:
+ * Thomas Miric - marv-t
+ *
+ * Jeti EX Bus Communication Protocol:
+ * http://www.jetimodel.com/en/show-file/642/
+ *
+ * JETI Telemetry Communication Protocol:
+ * http://www.jetimodel.com/en/show-file/26/
+ *
+ * Following restrictions:
+ * Communication speed: 125000 bps
+ * Number of channels: 16
+ *
+ * Connect as follows:
+ * Jeti EX Bus -> Serial RX (connect directly)
+ * Serial TX -> Resistor(2k4) ->Serial RX
+ * In jeti pdf it is different, but if the resistor breaks, the receiver continues to operate.
+ *
+ */
+
+#include <stdbool.h>
+#include <stdlib.h>
+
+#include "platform.h"
+#include "build_config.h"
+#include "drivers/system.h"
+#include "drivers/serial.h"
+#include "drivers/serial_uart.h"
+#include "io/serial.h"
+#include "rx/jetiexbus.h"
+
+
+#ifdef TELEMETRY
+
+#include <string.h>
+#include "sensors/sensors.h"
+#include "sensors/battery.h"
+#include "sensors/barometer.h"
+#include "telemetry/telemetry.h"
+
+#endif //TELEMETRY
+
+#include "rx/rx.h"
+
+//
+// Serial driver for Jeti EX Bus receiver
+//
+#define JETIEXBUS_BAUDRATE 125000                           // EX Bus 125000; EX Bus HS 250000 not supported
+#define JETIEXBUS_OPTIONS (SERIAL_STOPBITS_1 | SERIAL_PARITY_NO | SERIAL_NOT_INVERTED )
```

Comment:
Yes, this is possible. I have tested it with the port option SERIAL_BIDIR, but you need a pullup resistor to 3.3V. The internal pullups are not available in this mode. It's easier to connect RX and TX pins.